jwtproxy/src/server.py

```diff
@@ -13,7 +13,7 @@
 from yarl import URL
 
 
-LOG_LEVEL = env.get("LOG_LEVEL", logging.INFO)
+LOG_LEVEL = env.get("LOG_LEVEL", "DEBUG")
```

Comment:
this means that if log_level environment var does not exist we switch default to debug?